go/saltpack/decrypt.go

```diff
@@ -132,7 +148,7 @@ func (ds *decryptStream) readBlock(b []byte) (n int, lastBlock bool, err error)
 	return n, false, err
 }
 
-func (ds *decryptStream) tryVisibleReceivers(hdr *EncryptionHeader, ephemeralKey BoxPublicKey) (BoxSecretKey, BoxPrecomputedSharedKey, *SymmetricKey, int, error) {
+func (ds *decryptStream) tryVisibleReceivers(hdr *EncryptionHeader, ephemeralKey BoxPublicKey) (BoxSecretKey, *SymmetricKey, int, error) {
```

Comment:
Previously we needed to compute a Box for each payload packet, as part of authentication. Now we compute an HMAC key once, and reuse it for each payload packet. There's no need to save the `beforenm` shared secret anymore, so I've removed it from the return signatures of these two functions.